.github/workflows/synchronize-resources.yml

```diff
@@ -9,7 +9,10 @@ on:
     paths:
       - 'Sources/AirbnbSwiftFormatTool/airbnb.swiftformat'
       - 'Sources/AirbnbSwiftFormatTool/swiftlint.yml'
-      - .github/workflows/synchronize-resources.yml
+      - '.github/workflows/synchronize-resources.yml'
+      - '.pre-commit-config.yaml'
+      - 'scripts/git-format-staged.sh'
+      - 'scripts/swiftformat-wrapper.sh'
```

Comment:
We need to manually list these files to be copied as well. Take a look at lines 32:35, and 38:41

Reply:
right, sorry for overlooking